tests/CMakeLists.txt

```diff
@@ -88,6 +88,10 @@ function(add_unit_test name)
     target_compile_options(${name} PRIVATE -fblocks)
     target_compile_options(${name} PRIVATE -Wall -Wno-deprecated-declarations)
   endif()
+  if(NOT BUILD_SHARED_LIBS AND CMAKE_SYSTEM_NAME STREQUAL Android AND
+     CMAKE_SYSTEM_PROCESSOR STREQUAL armv7-a)
+    target_link_options(${name} PRIVATE "LINKER:--allow-multiple-definition")
+  endif()
```

Comment:
I think this needs a huge comment.  This is indicative of an issue - the multiple definitions shouldnt be there.  The warnings indicate that there is a mix up between libgcc and libunwind, which is a really bad thing as it will crash at runtime in a difficult to diagnose manner (the internal layouts of the unwind context is not compatible/guaranteed).

Reply:
Yeah, I don't know exactly what's going on here, [their doc doesn't clear it up for me](https://android.googlesource.com/platform/ndk/+/master/docs/BuildSystemMaintainers.md#Unwinding). All I know is that applying this workaround flag gets the test executables to build, and I tried moving a handful to Android and they ran fine. I will add a comment.

Reply:
Added the comment.